server/tools/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@peertube/remote",
```

Comment:
@peertube/cli no?

Reply:
Since remote tools are going to get published and not really the local tools, I felt like there needed some disambiguation here. Anyway it's not like we are publishing the tool right now, we can still change the name :slightly_smiling_face: